### PR TITLE
Remove unused github.com/thalesignite/crypto11 dep

### PR DESCRIPTION
**Pull request by bassosimone**
```
$ dep ensure
Warning: the following project(s) have [[constraint]] stanzas in Gopkg.toml:

  ✗  github.com/thalesignite/crypto11

However, these projects are not direct dependencies of the current project:
they are not imported in any .go files, nor are they in the 'required' list in
Gopkg.toml. Dep only applies [[constraint]] rules to direct dependencies, so
these rules will have no effect.

Either import/require packages from these projects so that they become direct
dependencies, or convert each [[constraint]] to an [[override]] to enforce rules
on these projects, if they happen to be transitive dependencies.
```

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -45,7 +45,7 @@
   branch = "master"
   name = "github.com/miekg/pkcs11"
 
-[[constraint]]
+[[override]]
   branch = "master"
   name = "github.com/thalesignite/crypto11"
```
Comment (hellais) [inline Gopkg.toml]: Ah yes this was a dependency for the code signing code which was removed from the master branch.

Maybe we can actually just drop it even as an override directive?

Comment (bassosimone) [reply Gopkg.toml]: Done!

**Comment by hellais** [comment]
lgtm

**Comment by bassosimone** [comment]
Thanks!

**Pull request merged**